table/block_based/partitioned_filter_block.cc

```diff
@@ -285,6 +308,79 @@ bool PartitionedFilterBlockReader::MayMatch(
       lookup_context);
 }
 
+void PartitionedFilterBlockReader::MayMatch(
+    MultiGetRange* range, const SliceTransform* prefix_extractor,
+    uint64_t block_offset, bool no_io, BlockCacheLookupContext* lookup_context,
+    FilterManyFunction filter_function) const {
+  CachableEntry<Block> filter_block;
+  Status s = GetOrReadFilterBlock(no_io, range->begin()->get_context,
+                                  lookup_context, &filter_block);
+  if (UNLIKELY(!s.ok())) {
+    TEST_SYNC_POINT("FilterReadError");
```

Comment:
Change this and other instances to ```IGNORE_STATUS_IF_ERROR()``` like on line #282

Reply:
Merge. Nice catch.